src/PSR7/Exception/Validation/AddressValidationFailed.php

```diff
@@ -17,9 +17,9 @@ abstract class AddressValidationFailed extends ValidationFailed
     /**
      * @return static
      */
-    public static function fromAddrAndPrev(OperationAddress $address, Throwable $prev) : self
+    public static function fromAddrAndPrev(OperationAddress $address, ?Throwable $prev) : self
     {
-        $ex          = new static(sprintf('Validation failed for %s', $address), $prev->getCode(), $prev);
+        $ex          = new static(sprintf('Validation failed for %s', $address), $prev ? $prev->getCode() : 0, $prev);
```

Comment:
What the case of this? there is separater `fromAddr` constructor

Reply:
Reverted this change. This is indirectly used here:
https://github.com/lezhnev74/openapi-psr7-validator/blob/d232725aab943ca3ef58771b9cd441272f332d7f/src/PSR7/Validators/BodyValidator/MultipartValidation.php#L107